clang/test/CodeGenCXX/vtable-assume-load-nonzero-default-address-space.cpp

```diff
@@ -0,0 +1,288 @@
+// RUN: %clang_cc1 %s -triple=spirv64-unknown-unknown -fsycl-is-device -std=c++11 -emit-llvm -o %t.ll -O1 -disable-llvm-passes -fms-extensions -fstrict-vtable-pointers
```

Comment:
This test should be merged with the existing address-space.cpp test. I also noticed that the address space one has some missing changes compared to the baseline so ideally that would be merged with the base test, but can be done in a separate pull request.